### PR TITLE
[BACKPORT to v0.9] Replace Bintray repositories

### PR DESCRIPTION
**Pull request by rpudlowski93**
PR related to task with backport: #2166

### CODE REVIEW

File: core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/download-requirements.sh [hunk 1]
```diff
@@ -559,25 +559,15 @@ gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-PGDG
...
-enabled=1
-gpgkey=https://packagecloud.io/rabbitmq/erlang/gpgkey
-EOF
-)
-
 RABBITMQ_SERVER_REPO_CONF=$(cat <<'EOF'
-[rabbitmq_rabbitmq-server]
-name=rabbitmq_rabbitmq-server
+[rabbitmq-server]
+name=rabbitmq-rpm
```
Comment (mkyc) [inline core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/download-requirements.sh]: shouldn't those two values be the same for consistency?

File: core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/download-requirements.sh [hunk 2]
```diff
@@ -594,7 +584,6 @@ add_repo_as_file 'grafana' "$GRAFANA_REPO_CONF"
 add_repo_as_file 'kubernetes' "$KUBERNETES_REPO_CONF"
 add_repo_as_file 'opendistroforelasticsearch' "$OPENDISTRO_REPO_CONF"
 add_repo_as_file 'postgresql-10' "$POSTGRESQL_REPO_CONF"
-add_repo_as_file 'rabbitmq_erlang' "$RABBITMQ_ERLANG_REPO_CONF"
 add_repo_as_file 'rabbitmq_rabbitmq-server' "$RABBITMQ_SERVER_REPO_CONF"
```
Comment (mkyc) [inline core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/download-requirements.sh]:
```suggestion
add_repo_as_file 'shouldn't it be changed for consistency?' "$RABBITMQ_SERVER_REPO_CONF"
```

File: core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/redhat-7/download-requirements.sh [hunk 3]
```diff
@@ -589,25 +589,15 @@ gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-PGDG
...
-enabled=1
-gpgkey=https://packagecloud.io/rabbitmq/erlang/gpgkey
-EOF
-)
-
 RABBITMQ_SERVER_REPO_CONF=$(cat <<'EOF'
-[rabbitmq_rabbitmq-server]
-name=rabbitmq_rabbitmq-server
+[rabbitmq-server]
+name=rabbitmq-rpm
```
Comment (mkyc) [inline core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/redhat-7/download-requirements.sh]: consistency

File: core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/redhat-7/download-requirements.sh [hunk 4]
```diff
@@ -624,7 +614,6 @@ add_repo_as_file 'grafana' "$GRAFANA_REPO_CONF"
 add_repo_as_file 'kubernetes' "$KUBERNETES_REPO_CONF"
 add_repo_as_file 'opendistroforelasticsearch' "$OPENDISTRO_REPO_CONF"
 add_repo_as_file 'postgresql-10' "$POSTGRESQL_REPO_CONF"
-add_repo_as_file 'rabbitmq_erlang' "$RABBITMQ_ERLANG_REPO_CONF"
```
Comment (mkyc) [inline core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/redhat-7/download-requirements.sh:627]: consistency

**Review by mkyc — CHANGES_REQUESTED** [review]
Rather little consistency comments.

File: core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/download-requirements.sh (hunk 1 above)
Comment (rpudlowski93) [reply core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/download-requirements.sh]: Sure, fixed

**Review by atsikham — APPROVED** [review]

**Review by mkyc — APPROVED** [review]
👍

**Pull request merged**